Update authentication documentation for Notifications (#1468)
liveblocks/liveblocks: docs/pages/rooms/authentication/access-token-permissions/express.mdx

      configure permission access in an `api/liveblocks-auth` endpoint by
      creating the `liveblocks-auth.ts` file with the
      following code. This is where you will implement your security and
      define the rooms that the user has access to.

> The client will request a token either for a specific `room`, in which case the request body property room will be defined, or for resources outside a specific room (like notifications), in which case, `room` will be undefined.

Reviewer: ```suggestion
The client will request a token either for a specific `room`, in which case the request body property `room` will be defined, or for resources outside a specific room (like notifications), in which case, `room` will be undefined.
```

      configure permission access in an `api/liveblocks-auth` endpoint by
      creating the `liveblocks-auth.ts` file with the
      following code. This is where you will implement your security and
      define the rooms that the user has access to.

>       The client will either request a token for a specific room, in that case, the request body property `room` will be defined,
>       or for resources outside a specific room (Notifications for example), in that case, `room` will be undefined.

Reviewer 2: ```suggestion
The client will request a token either for a specific `room`, in which case the request body property room will be defined, or for resources outside a specific room (like notifications), in which case, `room` will be undefined.
```
